common/src/main/scala/akka/contrib/persistence/mongodb/MongoPersistence.scala

```diff
@@ -45,6 +51,14 @@ trait CanDeserializeJournal[D] {
   def deserializeDocument(document: D)(implicit serialization: Serialization, system: ActorSystem): Event
 }
 
+trait CanSuffixCollectionNames {
+  def getSuffixFromPersistenceId(persistenceId: String): String
+}
+
```

Comment:
Can you name `SuffixCollectionNames` something that makes it more obvious that it's the default behavior that squashes all to an empty suffix?